server/handlers.go

```diff
@@ -134,8 +134,9 @@ func AuthHandler(w http.ResponseWriter, r *http.Request) {
 	case "verbose":
 		VerboseHandler(w, r)
 	case "list":
-		// TODO: check whether parameter is there or not....
-		ListHandler(w, r, strings.Split(r.URL.RawQuery, "=")[1])
+		ListHandler(w, r, r.URL.RawQuery)
+	case "action":
+		ActionHandler(w, r)
```

Comment:
@vkuznet I have added /action endpoint to perform actions on requests. It can also support bulk operations.
Test Command :

curl -X POST http://localhost:8989/action -d '[{"id":"123","action":"delete"},{"id":"789","action":"transfer"}]'
My question is: How do you want to handle this load? There can be single operation or bulk operations.